Content.Shared/_RMC14/Xenonids/Construction/SharedXenoConstructionSystem.cs

```diff
@@ -417,7 +421,7 @@ private void OnActionConstructionChosen(Entity<XenoChooseConstructionActionCompo
             _prototype.HasIndex(args.Choice))
         {
             action.Icon = new SpriteSpecifier.EntityPrototype(args.Choice);
-            Dirty(xeno, action);
+            Dirty(xeno);
```

Comment:
Xeno and xeno action are different, one's for the construction action comp and the other is for the action data comp